### ISSUE TITLE
Update to recent Magpie version

### ISSUE THREAD

**Issue by huard** [Ouranosinc/PAVICS-DataCatalog#50]

**Comment by davidcaron** [comment]
So I fixed a magpie issue, and I started working on this in this branch: https://github.com/Ouranosinc/PAVICS-DataCatalog/tree/update-magpie-and-docker

As a general rule, I assume that this repo will not be used heavily in the future, and I will not make significant updates to it (eg: update to python 3, even if dependencies are breaking if we don't pin their versions).

There is an issue I have, and it's related to solr, which I'm not familiar with at all. 

The error I get from solr is: "undefined field type". It doesn't mean that there is a `field type` that is not defined, but that there is a field named `type` that is not defined. The query sent to solr is the following:

http://192.168.99.201:8983/solr/birdhouse/select?start=0&rows=0&q=*&fl=*,score&fq=type:File&sort=id+asc&wt=json&indent=true

If I remove `fq=type:File`, the query seems to work (although I have no idea what I'm doing). This particular query parameter is hard coded in pypavics: https://github.com/Ouranosinc/pyPavics/blob/a521e2ba89d8c2b346d605675c35390da02a288c/pavics/catalog.py#L966

So I'm not sure where to go from here... @dbyrns or @huard any ideas?
